src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random RANDOM = new Random();
+    private int index = RANDOM.nextInt(Colors.values().length);
```

Comment:
the problem is that when you create `new ColorSupplier()` - this variable is initialized and is no further changed
so your number is not randomly generated each time you call the `getRandomColor()` method on the same ColorSupplier object - and this is a mistake

please move this variable to the method - so it could be reset each time you call the method